gcloud/orb.yaml

```diff
@@ -229,5 +229,95 @@ jobs:
           paths:
             - prev_version/<<parameters.zone>>
 
+  deploy-cloud-run:
```

Comment:
Lets make sure we add `description` fields to both new jobs and all the parameters within them. See the other jobs in this file for examples!

Wherever we're using the same parameters as in other jobs, lets make sure they match in terms of descriptions, defaults, etc.

Reply:
nit: lets keep the jobs in this file sorted

Reply:
ok cool, added descriptions and sorted all the jobs alphabetically